ape_hardhat/providers.py

```diff
@@ -522,7 +522,9 @@ def build_command(self) -> List[str]:
 
         return cmd
 
-    def reset_fork(self):
+    def reset_fork(self, block_number: int=None):
```

Comment:
```suggestion
    def reset_fork(self, block_number: Optional[int]=None):
```

Reply:
Ah yes this is better - learning a lot from you. Thank you so much! i added a commit!